docs/tutorials/brownian_motion.py

```diff
@@ -6,7 +6,7 @@
 #       extension: .py
 #       format_name: percent
 #       format_version: '1.3'
-#       jupytext_version: 1.16.4
+#       jupytext_version: 1.16.3
```

Comment:
Why downgrading

Reply:
Looks like the main branch has 1.16.3 pinned in the lock file.

Reply:
Please do so, thanks

Reply:
pin jupytext version in pyproject.toml 920cb74